File: src/at/jku/ssw/coco/intellij/CocoLanguage.java
package at.jku.ssw.coco.intellij;

import com.intellij.lang.Language;

/**
 * @author Thomas Scheinecker <a href="mailto:tscheinecker@gmail.com">tscheinecker@gmail.com</a>
 */
public class CocoLanguage extends Language {
    public static final CocoLanguage INSTANCE = new CocoLanguage();

    private CocoLanguage() {
        super("Cocol");
    }
}

File: src/main/java/at/scheinecker/intellij/coco/CocoLanguage.kt
package at.scheinecker.intellij.coco

import com.intellij.lang.Language

/**
 * @author Thomas Scheinecker [tscheinecker@gmail.com](mailto:tscheinecker@gmail.com)
 */
class CocoLanguage private constructor() : Language(ID) {
    companion object {
        const val ID = "Cocol/R"
        val INSTANCE = CocoLanguage()
    }
}
